src/components/welcome/welcome.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { render, waitFor } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import Welcome from './welcome'
+
+global.fetch = require('jest-fetch-mock')
+const apiStringResponse = 'Welcome to the Gilded Rose LLC store!'
+
+test('Welcome message is printed in the <Welcome /> component upon componentDidMount', async () => {
```

Comment:
Is it safe to say that I want my test statements to be somewhat from the user's perspective?

Reply:
Kind of. I would say that the test statements should test the business logic of your system. So for example, if you were building a set data structure using TDD, you might have tests like "if I try to add a duplicate element to the set, it does not do anything". Your end user may never ever see the set module that you create because it is many layers of abstraction below the web interface. So in that way, the end user does not care about the data structure per se.